### PR TITLE
Log hash of the beakerlib library repo

### PR DESCRIPTION
**Pull request by lukaszachy**
Make common tmt.utils.git_hash function to have single way of getting this information

Pull Request Checklist

* [x] implement the feature

### CODE REVIEW

File: tmt/utils.py [hunk 1]
```diff
@@ -4426,6 +4426,26 @@ def remove_color(text: str) -> str:
...
+
+    :param directory: path to a local git repository.
+    :param logger: used for logging.
+    :returns: short has as string
+    """
+    cmd = Command("git", "rev-parse", "--short", "HEAD")
+    result = cmd.run(cwd=directory, logger=logger)
+
+    if result.stdout is None:
+        raise RunError(message="Empty output.",
```
Comment (happz) [inline tmt/utils.py:4441]: How about "git returned no output when looking for hash of the current HEAD", or something similar? Without `TMT_SHOW_TRACEBACK=1`, this text, written in red, would be the only error message emitted by tmt, and "Empty output" does not really say much about where to start digging.

Comment (lukaszachy) [reply tmt/utils.py]: changed in eb63e98d

**Review by happz — APPROVED** [review]

**Review by psss — APPROVED** [review]
Thanks for improving this! Looks good, just fixed a typo with some minimal adjustments in bc5df8f0.

**Review by therazix — APPROVED** [review]

**Review by thrix — APPROVED** [review]
LGTM

**Pull request merged**